app/views/play/level/tome/SpellView.coffee

```diff
@@ -1102,7 +1102,7 @@ module.exports = class SpellView extends CocoView
     @spellHasChanged = true
 
   onAceMouseOut: (e) ->
-    Backbone.Mediator.publish("web-dev:stop-hovering-line")
+    Backbone.Mediator.publish("web-dev:stop-hovering-line", {})
```

Comment:
mediator.publish is basically a pub sub for backbone, correct?

Reply:
Yes, and the second parameter is the options argument, which gets validated; it was incorrect to be publishing these without any options, schema validation (which only runs in development) broke.